in_toto/util.py

```diff
@@ -103,7 +103,11 @@ def import_rsa_public_keys_from_files_as_dict(filepaths):
   key_dict = {}
   for filepath in filepaths:
     key = import_rsa_key_from_file(filepath)
-    keyid = key['keyid']
+    #FIXME: Add public key format check to ssl_crypto formats
+    if key["keyval"].get("private"):
```

Comment:
not sure if this is functionally equivalent but:

```python
if "private" in key["keyval"]:
    # blow up
```
May be better because it only checks for membership, it doesn't get the actual thing out of it